app/assets/javascripts/src/updateEvidenceType.js

```diff
@@ -0,0 +1,74 @@
+(function() {
+
+    'use strict';
+
+    var root = this, $ = root.jQuery;
+
+    if (typeof VOA === 'undefined') {
+        root.VOA = {};
+    }
+
+    var UpdateEvidenceType = function (){
+
+        var errorMessages = '<div id="error-summary" class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="govuk-error-summary">'+
+                '<h2 class="govuk-error-summary__title" id="error-summary-heading">There is a problem</h2>'+
+                '<ul class="govuk-list govuk-error-summary__list"><li></li></ul></div>';
+
+
+        var options = "input[value='lease'],input[value='license'],input[value='serviceCharge'],input[value='stampDutyLandTaxForm'],input[value='waterRateDemand'],input[value='otherUtilityBill'],input[value='landRegistryTitle'],input[value='unableToProvide']";
+
+        $(options).click(function() {
+          submitEvidenceType(this.value);
+        });
```

Comment:
Does this add the click event to all the inputs? If so, maybe there's a more concise selector that doesn't require knowing all the values? Maybe a class - or `input[type="radio"]`?